README.md

```diff
@@ -208,32 +208,33 @@ When running againts multiple destination organizations, a seperate working dire
 | dashboard_lists                        | Sync Datadog dashboard lists.                            |
-| logs_pipelines                         | Sync Datadog logs OOTB integration and custom pipelines. |
-| logs_pipelines_order                   | Sync Datadog logs pipelines order.                       |
-| logs_custom_pipelines (**deprecated**) | Sync Datadog logs custom pipelines.                      |
-| notebooks                              | Sync Datadog notebooks.                                  |
+| dashboards                             | Sync Datadog dashboards.                                 |
+| downtime_schedules                     | Sync Datadog downtimes.                                  |
+| downtimes (**deprecated**)             | Sync Datadog downtimes.                                  |
 | host_tags                              | Sync Datadog host tags.                                  |
+| logs_custom_pipelines (**deprecated**) | Sync Datadog logs custom pipelines.                      |
 | logs_indexes                           | Sync Datadog logs indexes.                               |
 | logs_indexes_order                     | Sync Datadog logs indexes order.                         |
 | logs_metrics                           | Sync Datadog logs metrics.                               |
+| logs_pipelines                         | Sync Datadog logs OOTB integration and custom pipelines. |
+| logs_pipelines_order                   | Sync Datadog logs pipelines order.                       |
 | logs_restriction_queries               | Sync Datadog logs restriction queries.                   |
 | metric_tag_configurations              | Sync Datadog metric tags configurations.                 |
+| metrics_metadata                       | Sync Datadog metric meradata.                            |
+| monitors                               | Sync Datadog monitors.                                   |
+| notebooks                              | Sync Datadog notebooks.                                  |
+| powerpacks                             | Sync Datadog powerpacks.                                 |
 | restriction_policies                   | Sync Datadog restriction policies.                       |
+| roles                                  | Sync Datadog roles.                                      |
+| service_level_objectives               | Sync Datadog SLOs.                                       |
+| slo_corrections                        | Sync Datadog SLO corrections.                            |
+| spans_metrics                          | Sync Datadog spans metrics.                              |
+| synthetics_global_variables            | Sync Datadog synthetics global variables.                |
+| synthetics_private_locations           | Sync Datadog synthetics private locations.               |
+| synthetics_tests                       | Sync Datadog synthetics tests.                           |
```

Comment:
```suggestion
| synthetics_global_variables            | Sync Datadog synthetic global variables.                |
| synthetics_private_locations           | Sync Datadog synthetic private locations.               |
| synthetics_tests                       | Sync Datadog synthetic tests.                           |
```

Reply:
Thanks! Will update this in the other PR